src/index.ts

```diff
@@ -1,22 +1,28 @@
-import assert from 'assert';
-import ExtendableError from 'es6-error';
-
+import * as assert from 'assert';
+import ExtendableError from 'extendable-error';
 const isString = d => Object.prototype.toString.call(d) === '[object String]';
 const isObject = d => Object.prototype.toString.call(d) === '[object Object]';
-
+interface ErrorConfig {
+  message: string;
+  time_thrown: any;
+  data: any,
+  options: any,
```

Comment:
Should we explicitly type the options?  That seems like a good idea?

Reply:
Yes, why not?